subprojects/docs/src/snippets/buildCache/cacheable-bundle-task/groovy/build.gradle

```diff
@@ -7,10 +7,17 @@ abstract class NpmTask extends DefaultTask {
 
     ListProperty<String> args = project.objects.listProperty(String)
 
+    @Inject
+    abstract ProjectLayout getProjectLayout()
+
     @TaskAction
     void run() {
-        project.file("$project.buildDir/bundle.js").withOutputStream { stream ->
-            project.file("scripts").listFiles().sort().each {
+        String bundleFilePath = "${projectLayout.buildDirectory.asFile.get()}/bundle.js"
+        File bundleFile = projectLayout.projectDirectory.file(bundleFilePath).asFile
+        File[] scriptsFiles = projectLayout.projectDirectory.file("scripts").asFile.listFiles()
```

Comment:
🔴 `dir("scripts")` would be more accurate, wouldn't it?

Reply:
Thanks for suggestion, fixed